serverless/README.md

```diff
@@ -121,7 +121,9 @@ To further configure your plugin, use the following custom parameters:
 | `nodeLayerVersion`          | Version of the Node.js Lambda layer to install, such as "29". Required if you are deploying at least one Lambda function written in Node.js and `addLayers` is true. Find the latest version number from [https://github.com/DataDog/datadog-lambda-js/releases][6].                                                                                                                                                                                                                                |
 | `dotnetLayerVersion`        | Version of the .NET Lambda layer to install, such as "14". Required if you are deploying at least one Lambda function written in .NET and `addLayers` is true. Find the latest version number from [https://github.com/DataDog/dd-trace-dotnet-aws-lambda-layer/releases][9].
 | `javaLayerVersion`        | Version of the Java Lambda layer to install, such as "12". Required if you are deploying at least one Lambda function written in Java and `addLayers` is true. Find the latest version number from [https://github.com/DataDog/datadog-lambda-java/releases][10].
-| `extensionLayerVersion`     | Version of the Datadog Lambda Extension layer to install, such as "5". When `extensionLayerVersion` is set, `apiKey` (or if encrypted, `apiKMSKey` or `apiKeySecretArn`) needs to be set as well. While using `extensionLayerVersion` do not set `forwarderArn`. Learn more about the Lambda extension [here][8].                                                                                                                                                                                   |
+| `extensionLayerVersion`     | Version of the Datadog Lambda Extension layer to install, such as "5". When `extensionLayerVersion` is set, `apiKey` (or if encrypted, `apiKMSKey` or `apiKeySecretArn`) needs to be set as well.                                                                                                                                                                                  |
+| `addExtension`                 | Whether to add the Lambda extension layer to the Lambda function. Defaults to true. When true, the extension layer is applied which handles sending logs, traces and custom metrics. The Parameter `extensionLayerVersion` must be set. While using `extensionLayerVersion` do not set `forwarderArn`.  Learn more about the Lambda extension [here][8].                                                                                                                 |
+| `exclude`                  | When set, the macro will ignore lambda specified functions. Use this if there are any functions that do not require datadog instrumentation. Defaults to [].                                                                               |
```

Comment:
Mmm where does `addExtension` come from?

Reply:
This was added in a previous commit. At the time I thought documentation was in a separate place. 😬 I am addressing it in this PR.